webpack.config.js

```diff
@@ -227,10 +227,11 @@ export default {
         }).join('\n');
       },
       override: {
-        'jquery.are-you-sure@*': {licenseName: 'MIT'},
+        'jquery.are-you-sure@*': {licenseName: 'MIT'}, // https://github.com/codedance/jquery.AreYouSure/pull/147
+        'khroma@*': {licenseName: 'MIT'}, // https://github.com/fabiospampinato/khroma/pull/33
       },
       emitError: true,
-      allow: '(Apache-2.0 OR BSD-2-Clause OR BSD-3-Clause OR MIT OR ISC OR CPAL-1.0)',
+      allow: '(Apache-2.0 OR BSD-2-Clause OR BSD-3-Clause OR MIT OR ISC OR CPAL-1.0 OR Unlicense)',
```

Comment:
Should we allow `Unlicense` packages come in?

Reply:
I see no issue, it's [essentially public domain](https://en.wikipedia.org/wiki/Unlicense). If we don't we have to drop mermaid:

```
gitea@
└─┬ mermaid@9.2.2
  └─┬ d3@7.6.1
    └─┬ d3-delaunay@6.0.2
      └─┬ delaunator@5.0.0
        └── robust-predicates@3.0.1
```